app/src/main/java/com/plaid/linksample/MainActivityJava.java

```diff
@@ -56,6 +60,26 @@
         return Unit.INSTANCE;
       }
   );
+  
+  // Experimental API using ActivityResultContract for androidx.fragment:1.3.0+
+  private ActivityResultLauncher<LinkTokenConfiguration> launchLinkActivity =
```

Comment:
Why aren't we breaking this out into its own activity? In the kotlin sample we show each one in a separate class. We'll probably keep both in the long run so developers don't have to rely on `androidx.fragment` if they don't want to, but it's probably easiest to digest if each file is a self contained single example with no extra code.

Reply:
Yeah agree with Zach here: we do support java as a first class citizen and hence should provide both ways of initializing our SDK.

I think we should consider deprecating our old `Plaid.create().initialize()` API somewhere in the next half year (align it's deprecation with public key deprecation):

1. Deprecate `Plaid.create().initialize()` API (next 6 months)
2. Swap examples and make `OpenPlaidLink` the default (Q3/Q4)
3. Remove our old API examples altogether (end of year)
4. Remove old API (???)